src/main/java/de/internetsicherheit/brl/bloxberg/cache/ArgParser.java

```diff
@@ -0,0 +1,44 @@
+package de.internetsicherheit.brl.bloxberg.cache;
+
+public class ArgParser {
+
+    public ArgParser() {
+
+    }
+    public String[] parseArgs(String[] args) {
```

Comment:
We should later refactor to return a CliParamter data object instead of a raw `String[]`, makes the interfaces in the other parts of the program much cleaner.